### PR TITLE
fix line endings in files that have mixed DOS/unix endings

### PR DESCRIPTION
**Pull request by AndrewMeadows**
Recent .gitattributes are causing git to automatically modify files with messed up line endings.  The intention of this PR is to reduce the pain and suffering.

### CODE REVIEW

File: libraries/gpu/src/gpu/Framebuffer.cpp [hunk 1]
```diff
@@ -7,279 +7,279 @@
 //
 //  Distributed under the Apache License, Version 2.0.
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
```
Comment (AndrewMeadows) [inline libraries/gpu/src/gpu/Framebuffer.cpp]: Line 9 and above were in **unix** format.  Lines 10 and below were in **DOS** format.  Insanity!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7fa069cf303b26a8fe72bddafafdb60c8355fac2 available at: https://gist.github.com/8005f3aa4ef74f4ad0d2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6123.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6123.dmg

**Comment by AndrewMeadows** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7fa069cf303b26a8fe72bddafafdb60c8355fac2 available at: https://gist.github.com/8005f3aa4ef74f4ad0d2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6123.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6123.dmg

**Pull request merged**